setup.py

```diff
@@ -29,8 +29,6 @@ def get_file_contents(file_path):
       url='http://www.privacyidea.org',
       packages=['privacyideautils'],
       install_requires=["requests",
-                        "pysqlite",
```

Comment:
We need pysqlite for the offline otp script, that is contained in the tools.

Reply:
actually we don't. `sqlite3` is enough and is shipped with python 2 and python 3.